src/std/sys.c

```diff
@@ -282,15 +304,24 @@ HL_PRIM int hl_sys_command( vbyte *cmd ) {
 #else
 	int status;
 	hl_blocking(true);
+#if TARGET_OS_IOS || TARGET_OS_TV
+	status = 0;
+	hl_error("hl_sys_command() not available on this platform");
+#else
 	status = system((pchar*)cmd);
+#endif
 	hl_blocking(false);
 	return WEXITSTATUS(status) | (WTERMSIG(status) << 8);
 #endif
 }
 
 HL_PRIM bool hl_sys_exists( vbyte *path ) {
+#if TARGET_OS_IOS || TARGET_OS_TV || __ANDROID__
+	return hl_mobile_file_exists(path);
+#else

```

Comment:
Same remarks as for fopen : no prefixing at HL level

Reply:
This one and other similar cases have been fixed in last commit.